docs/content/preview/architecture/docdb-sharding/tablet-splitting.md

```diff
@@ -211,15 +211,45 @@ For details on the architecture design, see [Automatic re-sharding of data with
 
-* In the high phase, each node has fewer than [`tablet_split_high_phase_shard_count_per_node`](../../../reference/configuration/yb-master/#tablet-split-high-phase-shard-count-per-node) shards. In this phase, YugabyteDB splits tablets larger than [`tablet_split_high_phase_size_threshold_bytes`](../../../reference/configuration/yb-master/#tablet-split-high-phase-size-threshold-bytes).
+#### Low Phase
 
-* Once the shard count exceeds the high phase count, YugabyteDB splits tablets larger than [`tablet_force_split_threshold_bytes`](../../../reference/configuration/yb-master/#tablet-force-split-threshold-bytes).
+In the low phase, each node has fewer than [`tablet_split_low_phase_shard_count_per_node`](../../../reference/configuration/yb-master/#tablet-split-low-phase-shard-count-per-node) shards. In this phase, YugabyteDB splits tablets larger than [`tablet_split_low_phase_size_threshold_bytes`](../../../reference/configuration/yb-master/#tablet-split-low-phase-size-threshold-bytes).
+
+By default, a given table will be considered to be in the low phase until there are 8 tablets per node. In this phase, a tablet will be split once it reaches 512 MB in size.
+
+#### High Phase
+
+In the high phase, each node has fewer than [`tablet_split_high_phase_shard_count_per_node`](../../../reference/configuration/yb-master/#tablet-split-high-phase-shard-count-per-node) shards. In this phase, YugabyteDB splits tablets larger than [`tablet_split_high_phase_size_threshold_bytes`](../../../reference/configuration/yb-master/#tablet-split-high-phase-size-threshold-bytes).
+
+By default, a table will be considered to be in the high phase until there are 24 tablets per node. In this phase, a tablet will be split once it reaches 10 GB in size.
+
+#### Final Phase
+
+Once the shard count exceeds the high phase count, YugabyteDB splits tablets larger than [`tablet_force_split_threshold_bytes`](../../../reference/configuration/yb-master/#tablet-force-split-threshold-bytes). This will continue until we reach the [`tablet_split_limit_per_table`](../../../reference/configuration/yb-master/#tablet-split-limit-per-table) tablets per table limit (if set to 0, there will be no limit).
+
+By default, a given table will be considered in this final phase once there are more than 24 tablets per node (determined by `tablet_split_high_phase_shard_count_per_node`). Once in this phase, a tablet will be split once it reaches 100 GB in size. Splits can continue in this phase until there are 256 tablets for this table.
+
+### Post-split compactions
+
+Once a split has been executed on a tablet, the two resulting tablets will require a full compaction in order to remove unnecessary data from each. These post-split compactions can require fairly significant CPU and disk resources, and impact workload performance as a result. To limit this impact, the number of simultaneous tablet splits allowed is conservative by default.
```

Comment:
Can shorten "can require fairly significant CPU and disk resources, and impact workload performance as a result". E.g to "can significantly increase CPU and disk usage and thus impact performance".